src/efi.rs

```diff
@@ -381,10 +382,42 @@ impl Component for Efi {
     }
 
     fn generate_update_metadata(&self, sysroot_path: &str) -> Result<ContentMetadata> {
-        let ostreebootdir = Path::new(sysroot_path).join(ostreeutil::BOOT_PREFIX);
         let dest_efidir = component_updatedir(sysroot_path, self);
 
-        if ostreebootdir.exists() {
+        // New EFI dir /usr/lib/efi
```

Comment:
Do you know if this this specific to the new Fedora grub/shim packages? I suspect it is...we may want to somehow make this a build time or even runtime conditional so in theory it's more pluggable/controllable by others.

I'd at least factor it out into a `const` somewhere that explains where it came from.

Hmm actually, this topic also strongly relates to https://github.com/coreos/bootupd/issues/766 right?